app/Helpers/render/forum.php

```diff
@@ -43,7 +43,7 @@ function RenderRecentForumPostsComponent(int $numToFetch = 4): void
             echo "<div class='embedded mb-1'>";
             echo "<div class='flex justify-between items-center'><div>";
             echo userAvatar($author, iconSize: 16);
-            echo " <span class='smalldate'>$postedAt</span></div>";
+            echo " <span class='smalldate cursor-help' title='" . Carbon::parse($nextData['PostedAt'])->format('F j Y, g:ia') . "'>$postedAt</span></div>";
```

Comment:
Tooltips are showing up when 'absolute dates' are enabled.
![image](https://github.com/RetroAchievements/RAWeb/assets/32680403/e9c8a80b-2e66-441d-b431-401e515f0cca)

Also in forum post history.

Reply:
Thanks, I figured I missed something. Should be resolved in https://github.com/RetroAchievements/RAWeb/pull/1652/commits/fb45f5af58d70395342162b89e197ec157b1f571.